### ISSUE TITLE
12.4.2 proportion_delayed is a bit misleading

### ISSUE THREAD

**Issue by leorjorge** [hadley/r4ds#1684]
Minor issue at https://r4ds.hadley.nz/logicals#sec-numeric-summaries-of-logicals. The explanation and variable name for proportion_delayed seems to be backwards from what is calculated, as it's giving the proportion of flights with less than 60 minute delay. Should probably be renamed proportion_ontime.

**Comment by mine-cetinkaya-rundel** [comment]
I agree that `proportion_delayed` alone doesn't explain this, though with the accompanying text it makes a bit more sense:

> the proportion of flights that were delayed on departure by at most an hour

Let's keep as is in order to avoid much longer variable names.

**Comment by leorjorge** [comment]
I'm sorry for disagreeing, but the way this variable is constructed, even flights with negative delay (departed early), will be counted towards a summary currently labeled `proportion_delayed`. This is extremely counter-intuitive... and the fact it's explained as 

> delayed on departure by at most an hour

 only makes the understanding of what the code is doing even more difficult.
